.travis.yml

```diff
@@ -1,18 +1,17 @@
 language: python
 sudo: false
-# We need ghostscript at least 9.15 and japanese minch fonts
-addons:
-    apt:
-        sources:
-        - debian-sid
-        packages:
-        - ghostscript
-        - fonts-takao-mincho
 env:
     - TOXENV=py27
+    - TOXENV=py33
+    - TOXENV=py34
 install:
     - travis_retry pip install tox
 script:
     - tox
 notifications:
     email: false
+before_install:
+    - wget http://downloads.ghostscript.com/public/binaries/ghostscript-9.15-linux-x86_64.tgz
+    - tar -xvf ghostscript-9.15-linux-x86_64.tgz
+    - mv ghostscript-9.15-linux-x86_64/gs-915-linux_x86_64 ghostscript-9.15-linux-x86_64/gs
+    - export PATH=$PWD/ghostscript-9.15-linux-x86_64:$PATH
```

Comment:
Well, I was able to make travis work with packaged (red part of this diff) just a week ago, in #36. It's a bit of hack too, because it relies on debian-sid packages, so I'm not completely sure which solution is better. At least packaged version will raise alarms if ghostscript will become incompatible.

Test technically passes without japanese fonts, but it renders blank page PDF, and compares it to the same blank page. Images of these pages obviously match, but the test is effectively skipped, as it doesn't test anything.